docs/source/markdown/podman-commit.1.md

```diff
@@ -38,10 +38,6 @@ Can be set multiple times
 Set the format of the image manifest and metadata.  The currently supported formats are _oci_ and _docker_.  If
 not specifically set, the default format used is _oci_.
 
-**--iidfile**=*ImageIDfile*
-
-Write the image ID to the file.
```

Comment:
We should probably support this. @edsantiago Mind opening up an issue so we remember to add this?

Reply:
Done: #5461